packages/ner/src/extractor-trim.js

```diff
@@ -54,6 +54,13 @@ class ExtractorTrim {
   matchBetween(utterance, condition, name) {
     const result = [];
     let matchFound;
+    if(typeof condition.regex === 'string') {
```

Comment:
Please stick to code style :-)

```suggestion
    if (typeof condition.regex === 'string') {
```